azure-pipelines.yml

```diff
@@ -30,15 +30,27 @@ stages:
       vmImage: 'ubuntu-latest'
     strategy:
       matrix:
-        py3.8_wag3:
+        py3.8_wag4:
           PYTHON_VERSION: '3.8'
-          WAGTAIL: '3.*'
+          WAGTAIL: '4.*'
+        py3.8_wag5:
+          PYTHON_VERSION: '3.8'
+          WAGTAIL: '5.*'
         py3.9_wag4:
           PYTHON_VERSION: '3.9'
           WAGTAIL: '4.*'
+        py3.9_wag5:
+          PYTHON_VERSION: '3.9'
+          WAGTAIL: '5.*'
+        py3.10_wag4:
+          PYTHON_VERSION: '3.10'
+          WAGTAIL: '4.*'
         py3.10_wag5:
           PYTHON_VERSION: '3.10'
           WAGTAIL: '5.*'
+        py3.11_wag4:
+          PYTHON_VERSION: '3.11'
+          WAGTAIL: '4.*'
```

Comment:
- Dropped Wagtail 3.x: https://endoflife.date/wagtail
- Make the python-wagtail test pipelines complete